peerpod-ctrl/go.mod

```diff
@@ -131,3 +131,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/confidential-containers/cloud-api-adaptor => ../
```

Comment:
We can remove this `replace` directive if we apply the change I mentioned above.

Reply:
As mentioned above this still results in the linting tools failing: https://github.com/confidential-containers/cloud-api-adaptor/actions/runs/5209437464/jobs/9399283534?pr=1059 :(